.github/workflows/release-legacy.yml

```diff
@@ -0,0 +1,29 @@
+name: Release legacy docs to gh-pages.
+on: workflow_dispatch
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    name: Deploy docs
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          repo-token: ${{ secrets.BENCE_TOPPANTO_ACCESS_TOKEN_FOR_LEGACY_DEPLOY }}
```

Comment:
I think we should change the name of this secret.
`LEGACY_DOCS_ACCESS_TOKEN`, what do you think?

Reply:
It's a really personal token, bound to my personal account, so I think my name should be present in the name